packages/ui/src/components/DateInput/components/CalendarContent.tsx

```diff
@@ -0,0 +1,107 @@
+import styled from '@emotion/styled'
+import { useContext } from 'react'
+import { Button } from '../../Button'
+import { Stack } from '../../Stack'
+import { Text } from '../../Text'
+import { DateInputContext } from '../Context'
+import { getNextMonth, getPreviousMonth } from '../helpers'
+import { Daily } from './CalendarDaily'
+import { Monthly } from './CalendarMonthly'
+
+const CapitalizedText = styled(Text)`
+  display: inline-block;
```

Comment:
not sure this is needed as you're using a span it should already be inline. But I'm not 100% sure, you need to check haha